src/containers/forms/CreateCategoryForm.jsx

```diff
@@ -42,12 +45,37 @@ class CreateCategoryForm extends Component {
   static propTypes = {
     closeModal: PropTypes.func.isRequired,
     createCategory: PropTypes.func.isRequired,
+    editCategory: PropTypes.func,
     message: PropTypes.shape({
       type: PropTypes.string,
       text: PropTypes.string,
     }),
+    selectedItem: PropTypes.shape({ id: PropTypes.string }),
   };
 
+  static getDerivedStateFromProps = (nextProps, state) => {
+    const { selectedItem } = nextProps;
+    if (selectedItem.id) {
+      const {
+        value,
+        name,
+        description,
+        supportsMultipleParticipants,
+      } = selectedItem;
+      const formTitle = 'Edit Category Request Form';
+      const btnText = 'Update';
+      return {
+        value: value.toString(),
+        name,
+        description,
+        supportsMultiple: supportsMultipleParticipants,
```

Comment:
@Chris-Maina just curious is there a reason you're not returning `supportsMultipleParticipants` here but assigning it to `supportsMultiple`.

Reply:
@andela-fomokaro I found out that the component was using the state `supportsMultiple ` and the payload i.e. selectedItem prop had `supportsMultipleParticipants `.